compiler/integration-tests/circuits/assert_lt/src/main.nr

```diff
@@ -0,0 +1,13 @@
+use dep::std;
+
+fn main(x: u64, y: pub u64) -> pub u64 {
+    // We include a println statement to show that noirJS will ignore this and continue execution
+    std::println("foo");
+
+    // A dynamic assertion message is used to show that noirJS will ignore the call and continue execution
+    // The assertion passes and thus the foreign call for resolving an assertion message should not be called.
```

Comment:
I think we can technically update this now that errors are part of the ABI :)! This is non-blocking I'm just noting it